lib/pbench/cli/server/shell.py

```diff
@@ -110,7 +110,16 @@ def run_gunicorn(server_config: PbenchServerConfig, logger: Logger) -> int:
     except OpenIDClient.NotConfigured as exc:
         logger.warning("OpenID Connect client not configured, {}", exc)
         notifier.notify("STOPPING=1")
+        notifier.notify("STATUS=OPENID broker not configured")
+        return 1
+    except [
+        OpenIDClient.ServerConnectionError,
+        OpenIDClient.ServerConnectionTimedOut,
+    ] as exc:
```

Comment:
A tuple would be better (and more "conventional") here than a list; not a big deal, but it looks a bit unusual! (Not to mention that the wrapping makes me want to shorten those names or remove the prefixes...)